Specs/Number/Swedish/PercentModel.json

```diff
@@ -0,0 +1,138 @@
+[
+  {
+    "Input": "100%",
+    "Results": [
```

Comment:
For any new specs for unsupported types under implementation in .NET, we should add:
"NotSupportedByDesign": "javascript, python, java",
and for the cases not currently runnable on .NET, "NotSupported": "dotnet". This avoid unintended bug breaks in other platforms and a little easier tracking of what is still unsupported.
I'm OK with merging this PR as-is as it doesn't cause any breaks, but in the future PRs could you add these annotations?